static/js/panel.js

```diff
@@ -1,147 +1,106 @@
-$(function () {
-  /**
-   * Toggles the ARIA expanded and hidden attributes' state.
-   *
-   * @param Object elem The element on which to toggle the attribute.
-   */
-  function toggleAria(elem) {
-    var expandedState =
-      elem.attr("aria-expanded") === "true" ? "false" : "true";
-    var hiddenState = elem.attr("aria-hidden") === "true" ? "false" : "true";
-    elem.attr("aria-hidden", hiddenState);
-    elem.attr("aria-expanded", expandedState);
-  }
-
-  $("#panel-toggle").click(function (event) {
-    var panelContent = $("#panel-content");
-    var icon = $(".navpanel-icon", this);
+var Panel = new (class Panel {
+  constructor() {
+    this.panel = document.getElementById("panel");
+    this.toggleButton = document.getElementById("panel-toggle");
+    this.icon = this.panel.querySelector(".navpanel-icon");
+    this.content = document.getElementById("panel-content");
+    this.accelEnabledCheckbox = document.getElementById("panel-accel-enable");
 
-    icon.toggleClass("expanded");
-    panelContent.toggle();
-    toggleAria(panelContent);
-  });
+    this.permalinkNode = this.findLink("Permalink");
+    this.logNode = this.findLink("Log");
+    this.rawNode = this.findLink("Raw");
 
-  var permalinkNode = $('.panel a[title="Permalink"]');
-  permalinkNode = permalinkNode.length == 0 ? null : permalinkNode[0];
+    this.toggleButton.addEventListener("click", () => this.toggle());
+    this.accelEnabledCheckbox.addEventListener("change", () => {
+      localStorage.setItem("accel-enable", event.target.checked ? "1" : "0");
+      this.updateAccelerators();
+    });
+    document.documentElement.addEventListener("keypress", event =>
```

Comment:
Note to self: verified on dev.searchfox.org that this still plays well with typeahead find (i.e. doesn't regress bug 1494348).